src/algebra/invertible.lean

```diff
@@ -168,6 +168,10 @@ inv_of_eq_right_inv (by simp)
 (is_unit_of_invertible (2:α)).mul_right_inj.1 $
   by rw [mul_sub, mul_inv_of_self, mul_one, bit0, add_sub_cancel]
 
+@[simp] lemma inv_of_two_add_inv_of_two [semiring α] [invertible (2 : α)] :
```

Comment:
I can't help mentioning that this actually holds for `non_assoc_semiring α` rather than just `semiring α` but I see `two_mul` is not sufficiently generalised so I wouldn't bother generalising here.